index.md

```diff
@@ -1,27 +1,13 @@
-# Your Title should go in the appropriate place in `index.html`, not here! Title is itself set only in the `<head>` element
 
-## Use headings only if you want to have subsections in your essay
 
 
 
-Add your content here, including _markup_ of **various kinds**. Now you can just write! But you should be aware of a couple of advanced features of markdown:
-- you can embed HTML directly n a markdown file. This is great if you want to add complex attributes like **classes**, e.g. to your `img` tags so that they display properly.
-  - another use of embedded HTML is for funky javascript function links. I've written a simple function that will seek to a particular time code in seconds.  You can call it like this: `<a href="javascript:seek(75)">seek to 1:15</a>`, but `[seek to 1:15](javascript:seek(75))` won't render properly, so you have to type the whole linke in, as seen here: <a href="#" onclick="javascript:seek(75)"> seek to 1:15</a>.
-  - also, Remarkable supports pandoc-style footnotes, which can be helpful sometimes. Here are some examples of footnotes at work:
 
-    Footnote 1 link[^first].
+ ## Interview With KCC President, Pema Bhutia
 
-    Footnote 2 link[^second].
+<body>
 
-    Inline footnote^[Text of inline footnote] definition.
+ <p>The documentation of culture through oral history is a unique way to understand the origins and practices of different societies.  Through the interview with Pema Bhutia of the Khangchendzonga Conservation Committee (KCC), I was able to retrieve new information each time I listened to the recording.  Bhutia discussed how tourism had socially, economically, and politically impacted the area.  One of the issues which I found most interesting was the environmental impact tourists were having on the land that conflicted with the daily activities of the locals, mentioned here, <a href="#" onclick="javascript:seek(75)"> seek to 7:50</a>.  For example, the availability of juniper berries on the mountain had increased the number of tourists harvesting.  These resources are used for incense in Buddhist practices.  The increase of tourists was accompanied by litter, the dumping of waste, and alterations in the landscape.  Holes made in the ground for tents and trees being cut down for fire wood were major issues.  The changes in the physical landscape had a negative impact on the spiritual practices of the locals who would use the mountain as a place for prayer and sanctuary.  These conditions are what led to the formation of the KCC in order to preserve the region and the culture’s traditions.</p>  
```

Comment:
If you test your links, you'll see that they're not linking to the right time location. Can you see what's wrong here? think a bit about how the `<a>` tag works in HTML.